rclcpp/include/rclcpp/node.hpp

```diff
@@ -148,11 +194,12 @@ class Node
     rclcpp::timer::CallbackType callback,
     rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
 
-  rclcpp::timer::WallTimer::SharedPtr
-  create_wall_timer(
-    std::chrono::duration<long double, std::nano> period,
-    rclcpp::timer::CallbackType callback,
-    rclcpp::callback_group::CallbackGroup::SharedPtr group = nullptr);
+  // TODO(wjwwood): reenable this once I figure out why the demo doesn't build with it.
```

Comment:
I got the demo to build with this function uncommented by changing https://github.com/ros2/demos/pull/12/files#diff-1c605fc584a4b2a7d84651a92fed1390R26 to:

```
timer_ = this->create_wall_timer(1.0_s, [this]() { ...
```

The reason it doesn't compile is because `1_s` is now ambiguous; the 1 could be an int or a double.

Even if you remove the definition of operator `"" _s` that takes a double, it doesn't compile with `1_s` specified to `create_wall_timer`, because there's no `"" _s` operator that returns nanoseconds, which is expected by `create_wall_timer`. And adding a function that returns nanoseconds and takes an integer would be ambiguous and also wouldn't compile (as you probably know from writing the functions added to `rclcpp.hpp` in this PR).

But, overall, I actually think it's fine to disable the `create_wall_timer` function that's commented out here, because it allows the user to specify times in sub-nanosecond precision, which is hard to achieve on most computing platforms...